include/picongpu/fields/incidentField/Functors.hpp

```diff
@@ -353,11 +353,14 @@ namespace picongpu
                             // In principle 1 ulp should work, but just to be safe against changes in unit system
                             constexpr double ulp = 4.0;
                             constexpr double eps = std::numeric_limits<double>::epsilon();
-                            bool const isMatchingUnit = (std::fabs(unitField[axis] - UNIT_EFIELD) <= eps * ulp);
+                            bool const isMatchingUnit
+                                = (std::fabs(unitField[axis] - UNIT_EFIELD) <= eps * UNIT_EFIELD * ulp);

```

Comment:
This fix should be pushed as a separate PR.

Reply:
It is already fixed in `dev` now.

Reply:
ok in this case a rebase is required.

Reply:
Done.